packages/@lwc/errors/package.json

```diff
@@ -2,6 +2,11 @@
     "name": "@lwc/errors",
     "version": "1.5.2",
     "description": "LWC Error Utilities",
+    "author": "LWC Team",
+    "repository": {
+        "type": "git",
+        "url": "git+https://github.com/salesforce/lwc.git"
+    },
```

Comment:
This is a public-private package so I don't think we want to make it easier for crawlers to find it.